README.md

```diff
@@ -182,8 +201,6 @@ Free Podcasts and Screencasts:
 
 ### Programming Playgrounds
 
-Write, compile and run your code within a browser. Try it out!
```

Comment:
did you mean to remove this?

Reply:
I think so too. Restore it according to #6819

Reply:
This was on accident. Just fixed it.